css/testimonial.css

```diff
@@ -1,762 +1,275 @@
-.testimonial{
-    /* Testimonial / 7 */
-
-
-width: 1440px;
-height: 896px;
-
-/* Brand / Turquoise 100 */
-
-background: #D5FAFC;
-
-/* Inside auto layout */
-
-flex: none;
-order: 3;
-flex-grow: 0;
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
 }
-.sectionheading{
-    /* Section Heading */
-
-
-/* Auto layout */
-
-display: flex;
-flex-direction: column;
-align-items: flex-start;
-padding: 0px;
-gap: 8px;
-
-position: absolute;
-width: 445px;
-height: 168px;
-left: 260px;
-top: 120px;
 
+.testimonial {
+    width: 1440px;
```

Comment:
Don't copy paste  the all css from figma. Always choose the relevant style for it. Here the width of the testimonial div should be 100% of the window size of your screen and also remove the top. It's not necessary here